pwndbg/gdblib/disasm/aarch64.py

```diff
@@ -97,6 +97,27 @@
     ARM64_EXT_SXTX: lambda x: bit_math.to_signed(x, 64),
 }
 
+AARCH64_MATH_INSTRUCTIONS = {
+    ARM64_INS_ADD: "+",
+    ARM64_INS_SUB: "-",
+    ARM64_INS_AND: "&",
+    ARM64_INS_ORR: "&",
+    ARM64_INS_ASR: ">>s",
+    ARM64_INS_ASRV: ">>s",
```

Comment:
I used `>>s` here to indicate arithmetic right shift and distinguish it from logical shift - as used here: https://msyksphinz-self.github.io/riscv-isadoc/html/rvi.html#srai